src/components/ReviewCard.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import * as api from "../api";
+import "./Reviews.css";
+
+function ReviewCard() {
+  const { review_id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [review, setReview] = useState();
+
+  useEffect(() => {
+    const fetchReviewData = async () => {
+      try {
+        const data = await api.fetchReview(review_id);
+        setReview(data);
+        setIsLoading(false);
+      } catch (error) {
+        setIsLoading(false);
+        console.error(`Error fetching review with ID ${review_id}:`, error);
```

Comment:
Remove console log
Also, this is the only catch you need, dont need the api ones